### ISSUE TITLE
Instructions should specify that vundle#end() needs to come before commands

### ISSUE THREAD

**Issue by chiphogg** [google/vim-codefmt#69]
The instructions read:

```vim
" Add maktaba and codefmt to the runtimepath.
" (The latter must be installed before it can be used.)
Plugin 'google/vim-maktaba'
Plugin 'google/vim-codefmt'
" Also add Glaive, which is used to configure codefmt's maktaba flags. See
" `:help :Glaive` for usage.
Plugin 'google/vim-glaive'
call glaive#Install()
" Optional: Enable codefmt's default mappings on the <Leader>= prefix.
Glaive codefmt plugin[mappings]
```

I do not think this will work, unless we stick a `call vundle#end()` between the last `Plugin` line, and the `call glaive#Install()`.  See chiphogg/vim-vtd#20 for an example of the problems this could cause.  (The associated commit also gives an example of how to improve the instructions.)

**Comment by malcolmr** [comment]
Though can't you also initialise Vundle with `call vundle#rc()` instead of `call vundle#begin()`, in which case `Plugin` runs immediately? (And I think that used to be the only option?)

But yes, perhaps just adding a `vundle#begin()` / `vundle#end()` pair in the instructions would clarify this?

**Comment by koalazen** [comment]
+1 on this, I thought it was a bug in my vimrc until I realized it works when placing the call outside of the vundle section.

**Comment by malcolmr** [comment]
Fixed by #70, I think.
